ReleaseChecklist.md

```diff
@@ -58,7 +58,7 @@
  - [ ] Create ``.release_ppa_auth`` at the root of your local Solidity checkout and set ``LAUNCHPAD_EMAIL`` and ``LAUNCHPAD_KEYID`` to your key's email and key id.
  - [ ] Double-check that the ``DISTRIBUTIONS`` list in ``scripts/release_ppa.sh`` and ``scripts/deps-ppa/static-z3.sh`` contains the most recent versions of Ubuntu.
  - [ ] Make sure the [``~ethereum/cpp-build-deps`` PPA repository](https://launchpad.net/~ethereum/+archive/ubuntu/cpp-build-deps) contains ``libz3-static-dev builds`` for all current versions of Ubuntu.
-       If not, run ``scripts/deps-ppa/static-z3.sh`` (after changing email address and key id) and wait for the builds to succeed before continuing.
+       If not, run ``scripts/deps-ppa/static-z3.sh`` (it will use the earlier created ``.release_ppa_auth`` file as well) and wait for the builds to succeed before continuing.
```

Comment:
If one wanted to nitpick, I guess this would be more idiomatic:
```suggestion
       If not, run ``scripts/deps-ppa/static-z3.sh`` (it will use the ``.release_ppa_auth`` file created earlier as well) and wait for the builds to succeed before continuing.
```
But fine either way :-).